minikubek8s.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Install kubectl
+echo "----------------------------kubectl checking----------------------------"
+export kubectl=$(which kubectl)
+if [ -f "$kubectl" ]; then
+    echo "$kubectl already exist...................................."
+else
+    echo "kubectl does not exist...................................."
+    echo "Installing kubectl........................................"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+    kubectl version
```

Comment:
What is the purpose of this `kubectl version`?

Reply:
just to show user kubectl already installed, and show the version :smile: